app/controllers/api/v1/shops_controller.rb

```diff
@@ -1,4 +1,8 @@
 class Api::V1::ShopsController < ApiController 
+  def index
+    render json: { shops: Shop.all }
```

Comment:
this is fine for now, but its time to start making serializers for your endpoints. Your next (non-styling) PR should have a serializer